app/views/layouts/_nav.html.erb

```diff
@@ -9,29 +9,19 @@
                 <!--
-                <li class="block lg:inline hover:font-bold hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]">
+                <li class="block lg:inline hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]">
                   <%= link_to "Industry", home_page_path, class: "block py-3 px-4 text-white", "aria-current": "page" %>
                 </li>
                 -->
-                <li class="block lg:inline hover:font-bold hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]">
-                  <%= link_to "Occupation", occupation_standards_path, class: "block py-3 px-4 text-white", "aria-current": "page" %>
-                </li>
-                <li class="block lg:inline hover:font-bold hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]">
-                  <%= link_to "State", state_standards_path, class: "block py-3 px-4 text-white", "aria-current": "page" %>
-                </li>
-                <li class="block lg:inline hover:font-bold hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]">
-                  <%= link_to "National", national_standards_path, class: "block py-3 px-4 text-white", "aria-current": "page" %>
-                </li>
+                <%= content_tag(:li, link_to("Occupations", occupation_standards_path, class: class_names("block py-3 px-4 text-white"), "aria-current": "page"), class: class_names("block lg:inline hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]", {active: current_page?(occupation_standards_path)})) %>
+                <%= content_tag(:li, link_to("State", state_standards_path, class: class_names("block py-3 px-4 text-white"), "aria-current": "page"), class: class_names("block lg:inline hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]", {active: current_page?(state_standards_path)})) %>
+                <%= content_tag(:li, link_to("National", national_standards_path, class: class_names("block py-3 px-4 text-white"), "aria-current": "page"), class: class_names("block lg:inline hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]", {active: current_page?(national_standards_path)})) %>
                 <li>
                   <span class="block w-0 h-0 md:h-6 md:w-px bg-seafoam-400"></span>
                 </li>
-                <li class="block border-t border-1 border-seafoam-400 md:border-none lg:inline hover:font-bold hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]">
-                  <%= link_to "Get Involved", home_page_path, class: "block py-3 px-4 text-white hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]" %>
-                </li>
+                <%= content_tag(:li, link_to("Get Involved", home_page_path, class: class_names("block py-3 px-4 text-white"), "aria-current": "page"), class: class_names("block lg:inline hover:shadow-[0_3px_0_0_rgba(249,111,0,1)]", {active: current_page?(home_page_path)})) %>
```

Comment:
Should we just make this path as "#" for now until we have something real for this page? Or maybe we should just set `active: false` for now. I think it looks a little weird to have both links highlighted when on the home page.

Reply:
I agree and have set active to false for now.